velox/core/QueryConfig.h

```diff
@@ -354,6 +354,15 @@ class QueryConfig {
   static constexpr const char* kDriverCpuTimeSliceLimitMs =
       "driver_cpu_time_slice_limit_ms";
 
+  /// The max bytes number of normalized keys for each row in the prefixsort.
```

Comment:
I left some comments for the documentation. Once these are addressed, let's update this file to match.